custom_components/tahoma/cover_devices/vertical_cover.py

```diff
@@ -98,18 +99,38 @@ def current_cover_position(self):
         # Uno devices can have a position not in 0 to 100 range when unknown
         if position is None or position < 0 or position > 100:
             return None
-
         return 100 - position
 
     async def async_set_cover_position(self, **kwargs):
         """Move the cover to a specific position."""
         position = 100 - kwargs.get(ATTR_POSITION, 0)
-        await self.async_execute_command(COMMAND_SET_CLOSURE, position)
+        if self.is_low_speed_enabled():
+            await self.async_execute_command(
+                COMMAND_SET_CLOSURE_AND_LINEAR_SPEED, position, "lowspeed"
+            )
+        else:
+            await self.async_execute_command(COMMAND_SET_CLOSURE, position)
 
     async def async_open_cover(self, **_):
         """Open the cover."""
-        await self.async_execute_command(self.select_command(*COMMANDS_OPEN))
+        if self.is_low_speed_enabled():
+            await self.async_execute_command(
+                COMMAND_SET_CLOSURE_AND_LINEAR_SPEED, 0, "lowspeed"
+            )
+        else:
+            await self.async_execute_command(self.select_command(*COMMANDS_OPEN))
 
     async def async_close_cover(self, **_):
         """Close the cover."""
-        await self.async_execute_command(self.select_command(*COMMANDS_CLOSE))
+        if self.is_low_speed_enabled():
+            await self.async_execute_command(
+                COMMAND_SET_CLOSURE_AND_LINEAR_SPEED, 100, "lowspeed"
+            )
+        else:
+            await self.async_execute_command(self.select_command(*COMMANDS_CLOSE))
+
+    def is_low_speed_enabled(self):
+        """Return if low speed mode is enabled."""
+        switch_entity_id = f"{self.entity_id.replace('cover', 'switch')}_low_speed"
```

Comment:
I am not a huge fan of this implementation, however this is most likely the easiest / fastest implementation. However I have seen that entity id's are not unique and if another integration tries to register the same one, it could be that you have something like `cover_low_speed_2` or something?

We could pull the entity id from registry based on the tahoma device id, on startup, however this is a more 'expensive' call.

Reply:
I got your point. Here I retrieve the current entity id of the cover I want to manipulate. So for sure, it will be unique. 
But you think, it can still be an issue?
If you have the Somfy API integration, and this one, you will have two covers: `cover.my_cover` and `cover.my_cover_2`. If the latest is from this integration, I will create a `switch.my_cover_2_low_speed`.

Reply:
In parallel, I will submit the same feature to the core within the official integration. I will see what they suggest :D

Reply:
Makes sense, let's keep it for now and we can always revisit in the future!